package.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+# Define the package metadata
+if [ -z "$VERSION" ]; then
```

Comment:
Change this to:
```
if [ -z "${VERSION:-}" ];
```
after adding the `set -eou pipefail`, else the script will exit with an 'unbound variable' error if VERSION is not set.